src/libs/ReportUtils.js

```diff
@@ -450,12 +451,29 @@ function getDefaultAvatar(login = '') {
     }
 
     // There are 24 possible default avatars, so we choose which one this user has based
-    // on a simple hash of their login
-    const loginHashBucket = hashLogin(login, CONST.DEFAULT_AVATAR_COUNT);
+    // on a simple hash of their login. Note that Avatar count starts at 1.
+    const loginHashBucket = hashLogin(login, CONST.DEFAULT_AVATAR_COUNT) + 1;
 
     return defaultAvatars[`Avatar${loginHashBucket}`];
 }
 
+/**
+ * Helper method to return the default avatar associated with the given login
+ * @param {String} [workspaceName]
+ * @returns {String}
+ */
+function getDefaultWorkspaceAvatar(workspaceName) {
+    if (!workspaceName) {
+        return defaultWorkspaceAvatars.WorkspaceBuilding;
+    }
+
+    // Remove all chars not A-Z or 0-9 including underscore
+    const withoutAccents = workspaceName.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
+    const alphaNumeric = withoutAccents.replace(/[^0-9a-z]/gi, '').toUpperCase();
```

Comment:
The `.replace(/[\u0300-\u036f]/g, '');` is unnecessary because later you do `.replace(/[^0-9a-z]/gi, '')` which would drop accents anyway.

```suggestion
    const alphaNumeric = workspaceName.normalize('NFD').replace(/[^0-9a-z]/gi, '').toUpperCase();
```